pyaedt/generic/configurations.py

```diff
@@ -1037,7 +1038,7 @@ def validate(self, config):
 
     @pyaedt_function_handler()
     def import_config(self, config_file, *args):
-        """Import configuration settings from a JSON file and apply it to the current design.
+        """Import configuration settings from a JSON or TOML file and apply it to the current design.
```

Comment:
If this docstring is too long, you could remove "current" given that it's understood that the method is working on the current/active design.